pages/mentor-listing.tsx

```diff
@@ -0,0 +1,64 @@
+import "twin.macro";
+import { Box, HStack, Grid, Text, Link } from "@chakra-ui/react";
+import { shuffle, times } from "lodash";
+import { X } from "phosphor-react";
+
+import { FilterSidebar, MentorCard } from "components/pages/MentorListing";
+import { NavBar } from "components/modules/NavBar";
+import { useScreen } from "hooks";
+
+/* -------------------------------------------------------------------------- */
+/*                                  Mock Data                                 */
+/* -------------------------------------------------------------------------- */
+
+const avatarPaths = shuffle(times(16))
+    .slice(0, 12)
+    .map((i) => `/svg/avatar-${i}.svg`);
+
+const resultCount = 32;
+
+/* -------------------------------------------------------------------------- */
+/*                               Mentor Results                               */
+/* -------------------------------------------------------------------------- */
+
+function Results() {
+    return (
+        <Box tw="h-screen w-full" overflow="scroll">
+            <Grid tw="justify-start items-center p-8 gap-x-4 gap-y-1">
+                <Text tw="text-xl inline font-medium">Search Results For “programming”</Text>
+                <Link tw="text-sm flex" href="/mentor" gridColumn={2}>
+                    <X size={20} /> Clear All Filters
+                </Link>
+                <div tw="text-secondary uppercase">{resultCount} Results</div>
+            </Grid>
+            <Grid
+                tw="gap-4 w-full items-center"
+                p={[4, 8]}
+                justifyItems="center"
+                justifyContent="center"
+                templateColumns="repeat(auto-fit, minmax(375px, 1fr))">
+                {avatarPaths.map((image) => (
+                    <MentorCard key={image} image={image} />
+                ))}
+            </Grid>
+        </Box>
+    );
+}
+
+/* -------------------------------------------------------------------------- */
+/*                             Mentor Listing Page                            */
+/* -------------------------------------------------------------------------- */
+
+export default function MentorListing() {
+    const { min } = useScreen();
+    return (
+        // Setting the height=100vh and overflow=hidden get rid of the scrollbar
+        <Box tw="h-screen overflow-hidden" background="trueGray.100">
+            <NavBar name="John" />
+            <HStack alignItems="start">
+                {min`sm` && <FilterSidebar />}
```

Comment:
Oh, we lost responsiveness here. Now mobile users cannot filter users. Jett made a button on the side of screen so that mobile users can use the filter feature.

Reply:
I've just seen your conversation with Jett now. Nvm. We will add the bottom bar for mobile version, right?

Reply:
Yeah. The previous solution was not fit for mobile so I'll putting that into our backlog.